_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -222,26 +305,87 @@ A set of two SVG `<a>` elements have the same [accessible name][] but link to di
 
 #### Failed Example 6
 
-Links resolves to same resource after redirect, but the redirect is not instant:
+Both links resolve to [same resource][] after redirect, but the redirect is not instant.
 
 ```html
 <a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a>
 <a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/redirect1.html">Contact us</a>
 ```
 
+#### Failed Example 7
```

Comment:
This is testing whether or not an implementor supports shadow DOM, not whether or not the rule is properly implemented. I think this should be removed. Failed example 8 covers this scenario well enough I think.